modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartQuickAdd.js

```diff
@@ -117,6 +117,63 @@ export default function CartQuickAdd() {
 
 				const {name, productConfiguration, urls} = parentProduct;
 
+				if (product.discontinued) {
```

Comment:
I don't get what is this discontinued field. I can't find it in the API definition

Reply:
I'm pretty sure it's a property on the product object that I'm working with here.

Reply:
Yes, it's a property on the product object:

<img width="679" alt="image" src="https://user-images.githubusercontent.com/24564752/230288525-43251a8d-2506-46d1-932b-283ba4b3768f.png">

Reply:
This is not a Product object, it is a SKU object